CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 
 All notable changes to this project will be documented in this file.
 
-## Fine Beta-2
+## Unreleased Sprint 60 Ending 2017-05-08
+
+### Fixed
+
+- Add notifications for finish_retirement. [(#106)](https://github.com/ManageIQ/manageiq-content/pull/106)
+- Change exceeds message in log to warn type. [(#104)](https://github.com/ManageIQ/manageiq-content/pull/104)
+- Change errors in log to error type. [(#101)](https://github.com/ManageIQ/manageiq-content/pull/101)
+- Changed ${/#ae_reason} to a string value in on_error methods. [(#98)](https://github.com/ManageIQ/manageiq-content/pull/98)
+- Add quota checking for VMReconfigure tests. [(#56)](https://github.com/ManageIQ/manageiq-content/pull/56)
```

Comment:
@mfeifer This was labeled as a `bug` but I would include it in the `Added` section as it brings in new functionality.